tlds/gen.go

```diff
@@ -0,0 +1,58 @@
+//+build ignore
+
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/alecthomas/template"
```

Comment:
Well that's weird, I thought I _was_ using `text/template`... not entirely sure where `goimports` got this from.